.buildconfig-android.yml

```diff
@@ -86,7 +86,7 @@ projects:
       type: aar
     description: Sync 1.5 remote tabs implementation.
   nimbus:
-    path: components/nimbus
+    path: components/nimbus/android
```

Comment:
This is just to make way for the iOS code.

Reply:
Just flagging that this is going to make conflicts with @dmose's work over in https://github.com/mozilla/application-services/pull/3932, I wonder if it's worth pulling this specific change out into a separate PR that we can land ASAP and both rebase on top of?

Reply:
Pulling this out, is a good idea.